Index: modules/libraries/jonas-deployment/src/main/java/org/ow2/jonas/deployment/common/JndiEnvRefsGroupDesc.java
===================================================================
--- modules/libraries/jonas-deployment/src/main/java/org/ow2/jonas/deployment/common/JndiEnvRefsGroupDesc.java	(revision 15742)
+++ modules/libraries/jonas-deployment/src/main/java/org/ow2/jonas/deployment/common/JndiEnvRefsGroupDesc.java	(working copy)
@@ -354,9 +354,11 @@
             }
 
             if (!resource.containsKey(rName)) {
-                throw new DeploymentDescException("jonas-resource missing for res-ref-name " + rName);
+                // FIXME  throw new DeploymentDescException("jonas-resource missing for res-ref-name " + rName);
+            	logger.log(BasicLevel.WARN, "jonas-resource missing for res-ref-name " + rName);
+            } else {
+            	resourceRefs.addElement(new ResourceRefDesc(classLoader, resourceRef, (JonasResource) resource.get(rName)));
             }
-            resourceRefs.addElement(new ResourceRefDesc(classLoader, resourceRef, (JonasResource) resource.get(rName)));
         }
     }
 
