Index: modules/management/src/main/java/org/mortbay/management/ObjectMBean.java
===================================================================
--- modules/management/src/main/java/org/mortbay/management/ObjectMBean.java	(revision 4062)
+++ modules/management/src/main/java/org/mortbay/management/ObjectMBean.java	(working copy)
@@ -139,7 +139,7 @@
                 }
                 catch (ClassNotFoundException e)
                 {
-                    if (e.toString().endsWith("MBean"))
+                    if (e.toString().indexOf("MBean") != -1)
                         Log.ignore(e);
                     else
                         Log.warn(e);
@@ -253,7 +253,7 @@
                             else if (key.indexOf('(')>0)
                             {
                                 // define an operation
-                                if (!defined.contains(key) && key.indexOf('[')<0)
+                                if (!defined.contains(key) && key.indexOf(']')<0)
                                 {
                                     defined.add(key);
                                     operations=LazyList.add(operations,defineOperation(key, value, bundle));
