Index: org/mortbay/jetty/plugin/AbstractJettyMojo.java
===================================================================
--- org/mortbay/jetty/plugin/AbstractJettyMojo.java	(revision 6087)
+++ org/mortbay/jetty/plugin/AbstractJettyMojo.java	(working copy)
@@ -29,6 +29,8 @@
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
 import org.apache.maven.project.MavenProject;
+import org.cipango.plugin.CipangoServer;
+import org.cipango.plugin.CipangoSipAppContext;
 import org.codehaus.plexus.util.FileUtils;
 import org.eclipse.jetty.security.LoginService;
 import org.eclipse.jetty.server.Connector;
@@ -50,7 +52,7 @@
     /**
      * A wrapper for the Server object
      */
-    protected JettyServer server;
+    protected CipangoServer server;
     
     /**
      * List of connectors to use. If none are configured
@@ -91,7 +93,7 @@
      * The "virtual" webapp created by the plugin
      * @parameter
      */
-    protected JettyWebAppContext webAppConfig;
+    protected CipangoSipAppContext webAppConfig;
 
 
 
@@ -332,12 +334,12 @@
     }
 
 
-    public JettyServer getServer ()
+    public CipangoServer getServer ()
     {
         return this.server;
     }
 
-    public void setServer (JettyServer server)
+    public void setServer (CipangoServer server)
     {
         this.server = server;
     }
@@ -385,7 +387,7 @@
             getLog().debug("Starting Jetty Server ...");
 
             printSystemProperties();
-            this.server = new JettyServer();
+            this.server = new CipangoServer();
             setServer(this.server);
 
             //apply any config from a jetty.xml file first which is able to
@@ -484,7 +486,7 @@
     {
         //As of jetty-7, you must use a <webAppConfig> element
         if (webAppConfig == null)
-            webAppConfig = new JettyWebAppContext();
+            webAppConfig = new CipangoSipAppContext();
         
         //Apply any context xml file to set up the webapp
         //CAUTION: if you've defined a <webAppConfig> element then the
