Index: org/mortbay/jetty/plugin/JettyRunMojo.java
===================================================================
--- org/mortbay/jetty/plugin/JettyRunMojo.java	(revision 6306)
+++ org/mortbay/jetty/plugin/JettyRunMojo.java	(working copy)
@@ -59,7 +59,6 @@
  *  </p>
  * 
  * 
- * @goal run
  * @requiresDependencyResolution runtime
  * @execute phase="test-compile"
  * @description Runs jetty directly from a maven project
